test/ModuleInterface/alwaysEmitConformanceMetadata_attr.swift

```diff
@@ -0,0 +1,17 @@
+// RUN: %empty-directory(%t)
+
+ // Ensure the attribute is printed in swiftinterface files
+ // RUN: %target-swift-frontend-typecheck -emit-module-interface-path %t/Foo.swiftinterface %s -module-name Foo
```

Comment:
Would be good to run the emitted interface through interface verification as well.

Reply:
You can use the pair of lit substitutions I created for this purpose:
```
// RUN: %target-swift-emit-module-interface(%t.swiftinterface) %s -module-name Foo
// RUN: %target-swift-typecheck-module-from-interface(%t.swiftinterface) -module-name Foo
```

Reply:
Done, thank you!